rocky-raft/src/rocky/raft/server/RaftServer.java

```diff
@@ -22,7 +23,8 @@
     private ServerLogic serverLogic;
 
     public RaftServer(int id) throws IOException {
-        LOG_TAG += id;
+        LOG_TAG += Config.serverAtIndex(id).getId();
```

Comment:
We should completely get rid of index IMHO. It might just complicate things. Why not just pass on the id from StartServer?